src/actions/toolbox.js

```diff
@@ -37,3 +37,19 @@ export function openWorkerToolbox(url: string) {
     }
   };
 }
+
+export function evaluateInConsole(inputString: string) {
+  return async ({ client, getState }: ThunkArgs) => {
+    const frameId = getSelectedFrameId(getState());
+
+    console.log(inputString);
+
+    client.evaluate(`${inputString}.toString()`, { frameId }).then(res => {
+      if (res.exception) {
+        const docLink = res.exceptionDocURL ? `\n${res.exceptionDocURL}` : "";
+        return console.error(res.exceptionMessage, docLink);
+      }
+      return console.log(res.result);
```

Comment:
i dont think we want the console.log here...

Reply:
I think I see two possible scenarios here, not sure which is being addressed `I dont think we want the _console.log_ here` or `I dont think we want the console.log _here_`